homeassistant/components/tuya/cover.py

```diff
@@ -68,6 +69,13 @@ class TuyaCoverEntityDescription(CoverEntityDescription):
             set_position=DPCode.PERCENT_CONTROL_3,
             device_class=DEVICE_CLASS_CURTAIN,
         ),
+        TuyaCoverEntityDescription(
```

Comment:
This is a non-documented DPCode. We should at least add a comment about that in here including a small note on what device this was based.

Reply:
Done.